packages/suite-analytics/src/types/events.ts

```diff
@@ -0,0 +1,377 @@
+import type { OnboardingAnalytics } from '.';
```

Comment:
This is cyclic import, you are importing from `types/index.ts` here but in same time you reexporing this file in that very same `types/index.ts` and that results in cyclic imports. Suite package is full of this stuff hidden under aliases and it's breaking chunk splitting and probably some other tools.

Get rid of index as I mentioned in comments above and this can't happen again.

Reply:
43bd592f480aa25c3ab04365431d3a83300da5a2